Makefile

```diff
@@ -64,7 +64,25 @@ run-local:
 	./meshery; \
 	cd ..
 
+run-tests:
+	# -----Linting Check-----
+	# GOPROXY=direct GOSUMDB=off go get github.com/mgechev/revive
```

Comment:
Remove the above test description & fetch the packages in the makefile itself.

Reply:
Yeah. I am a bit of pickle over here. Fetching them everytime was delaying the process a bit.
I thought of these options

1. Build an if-else case to check if the packages are already fetched. (Would bring more complexity because would need to customise it for different OS).

2. Make seperate `build-tests` kind of make argument for fetching the packages the first time.

3. Just include them with `run-tests` at cost of some delay, as you suggested.

I was more inclined to second option. Thoughts?

Reply:
> Just include them with run-tests at cost of some delay, as you suggested.

They wont be fetched everytime, after fetching once they will already be in go.mod so next time it won't be fetched again.